sdk/ai/azure-ai-projects/azure_ai_projects_tests.env

```diff
@@ -61,8 +61,8 @@ AZURE_AI_PROJECTS_TELEMETRY_TESTS_PROJECT_CONNECTION_STRING=${AZURE_AI_PROJECTS_
 ########################################################################################################################
 # Agents tests
 #
-AZURE_AI_PROJECTS_AGENTS_TESTS_PROJECT_CONNECTION_STRING=
-AZURE_AI_PROJECTS_AGENTS_TESTS_DATA_PATH=
+AZURE_AI_PROJECTS_AGENTS_TESTS_PROJECT_CONNECTION_STRING=eastus2.api.azureml.ms;1bd798d8-5940-47f2-81b4-b8389610f09c;ai-projects-sdk-testing-rg;ai-projects-sdk-testing-project-1
+AZURE_AI_PROJECTS_AGENTS_TESTS_DATA_PATH=azureml://subscriptions/1bd798d8-5940-47f2-81b4-b8389610f09c/resourcegroups/ai-projects-sdk-testing-rg/workspaces/ai-projects-sdk-testing-project-1/datastores/workspaceblobstore/paths/LocalUpload/99ccdcf221d468b60f565d5708f6ce1d/product_info_1.md
 AZURE_AI_PROJECTS_AGENTS_TESTS_STORAGE_QUEUE=
```

Comment:
This file should be left empty, don't update it. We have another version in the private repo that has the values.

Reply:
Fixed